index.html

```diff
@@ -565,10 +627,52 @@ <h2>Dictionary RTCQuicStreamEventInit Members</h2>
               <p>The <code><a>RTCQuicStream</a></code> object associated with the
               event.</p>
             </dd>
+            <dt><dfn><code>parameters</code></dfn> of type <span class=
+            "idlMemberType"><a>RTCQuicStreamParameters</a></span></dt>
+            <dd>
+              <p>The <code><a>RTCQuicStreamParameters</a></code> dictionary associated with the
+              event.</p>
+            </dd>
           </dl>
         </section>
       </div>
     </section>
+    <section id="rtcquicstreamdirection*">
+      <h3><dfn>RTCQuicStreamDirection</dfn> Enum</h3>
+      <p><code>RTCQuicStreamDirection</code> indicates the direction of the QUIC stream.</p>
+      <div>
+        <pre class="idl">enum RTCQuicStreamDirection {
+    "sendrecv",
+    "recv",
+    "send"
+};</pre>
+        <table data-link-for="RTCQuicStreamDirection" data-dfn-for="RTCQuicStreamDirection" class="simple">
+          <tbody>
+            <tr>
+              <th colspan="2">Enumeration description</th>
+            </tr>
+            <tr>
+              <td><dfn><code id="idl-def-RTCQuicStreamDirection.sendrecv">sendrecv</code></dfn></td>
+              <td>
+                <p>A bi-directional QUIC stream.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn><code id="idl-def-RTCQuicStreamDirection.recv">recv</code></dfn></td>
+              <td>
+                <p>A receive-only QUIC stream.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn><code id="idl-def-RTCQuicStreamDirection.send">send</code></dfn></td>
+              <td>

```

Comment:
In QUIC transport language, a uni-directional stream created by the local endpoint.

Reply:
Fixed.